libs/barista-components/breadcrumbs/src/breadcrumbs-item.ts

```diff
@@ -29,7 +29,7 @@ import { Directive, ElementRef } from '@angular/core';
     class: 'dt-breadcrumbs-item',
   },
 })
-export class DtBreadcrumbsItem2 {
+export class DtBreadcrumbsItem {
```

Comment:
I'm not sure we want to change the name as well? @ffriedl89 I guess the classname should stay the same right?